backend/Cargo.toml

```diff
@@ -28,7 +28,7 @@ rstar = "0.12.0"
 serde = "1.0.188"
 serde_json = "1.0.105"
 serde-wasm-bindgen = "0.6.0"
-utils = { git = "https://github.com/a-b-street/utils", features = ["serde"] }
+utils = { git = "https://github.com/a-b-street/utils", branch = "mkirk/update-geo", features = ["serde"] }
```

Comment:
Corresponds to https://github.com/a-b-street/utils/pull/3

Not sure how you want to sequence these merges, or if you care. LMK!

Reply:
Thanks for merging https://github.com/a-b-street/utils/pull/3 - I've removed this branch specification, which is no longer needed and updated Cargo.lock to the latest utils@main.

👍